lib/ae_mdw/names.ex

```diff
@@ -72,6 +72,22 @@ defmodule AeMdw.Names do
   @min_bin Util.min_bin()
   @max_bin Util.max_256bit_bin()
 
+  @spec count_names(state(), query()) :: {:ok, non_neg_integer()} | {:error, reason()}
+  def count_names(state, query) do
+    query
+    |> Util.convert_params(&convert_param/1)
+    |> case do
+      {:ok, %{owned_by: owned_by}} ->
+        {:ok, count_active_names(state, owned_by)}
+
+      {:ok, %{}} ->
+        {:ok, count_active_names(state, nil)}
```

Comment:
We can remove these two clauses by doing `{:ok, filters} -> count_active_names(state, Map.get(filters, :owned_by))`